server/services/emails/emailHelper.js

```diff
@@ -0,0 +1,292 @@
+const generateWeeklyEmailBody = responses => {
+  const individualResponse = () => {
+    let string = "";
+    for (let response of responses) {
+      string += `<div style="z-index: 4;
+    width: 560px;
+    height: 140px;
+    border-radius: 5px;
+    box-shadow: 0px 0px20px rgba(215,219,237,0.3);
+    margin: auto;
+    margin-bottom: 10px;
+    "
+    >
+    <img
+    src=${response.image}
+    alt=${response.pokemon}
```

Comment:
why is the alt response.pokemon

Reply:
omg 😲 thats wrong its supposed to be something else.